src/pages/Home/Content/Section/ItemList/ItemCard/ItemCardStyle.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const ItemCardStyle = styled.div`
```

Comment:
<img width="192" alt="image" src="https://user-images.githubusercontent.com/65440011/180611346-0f12ec3f-0f9d-4b33-8839-a63ca098ef18.png">
The gap in between the food name and price should be smaller than the space to the next card. Change the padding accordingly